package.json

```diff
@@ -42,7 +42,7 @@
     "postinstall": "node ./scripts/postinstall.js",
     "preinstall": "node ./scripts/use-yarn-please.js",
     "prelint": "yarn satisfied && yarn syncpack:list",
-    "publish:beachball": "beachball publish --scope \"!packages/fluentui/*\"",
+    "publish:beachball": "beachball publish",
```

Comment:
Seems like you should either get rid of this since it's not needed, or update it with appropriate shared args, like `beachball publish -b origin/master --access public -y`

Reply:
I've updated the script so that it can be used by both pipelines